User_Story#1/SortingApp/BusinessLayer/Utilities/DataReader.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System;
-using BusinessLayer.Utilities.Validator.Interfaces;
 using System.IO;
+using BusinessLayer.Utilities.Validator.Interfaces;
 
 namespace BusinessLayer.Reader
 {
```

Comment:
Since the Path property  is used in one methos, please move it as parameter in the method:
 ReadContent(string path)